### PR TITLE
Nas 104450

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/sharing/iscsi/iscsi.ts [hunk 1]
```diff
@@ -276,9 +276,7 @@ export const helptext_sharing_iscsi = {
     "Mutual secret password. Required when Peer User is set. Must be\
  different than the <i>Secret</i>."
   ),
-  authaccess_error_peersecret: T('Must match Peer Secret (Confirm) and be\
- between 12 and 16 characters in length. Cannot be the same as\
- Secret.'),
+  authaccess_error_peersecret: T('Secret and Peer Secret can not be the same.'),
```
Comment (a-stjohn) [inline src/app/helptext/sharing/iscsi/iscsi.ts]:
```suggestion
  authaccess_error_peersecret: T('Mutual secret password. Required when Peer User \
 is set. Must be different than the Secret.'),
```

Comment (lola27) [reply src/app/helptext/sharing/iscsi/iscsi.ts]: Thanks @a-stjohn , this one error message will be displayed under the secret field or the peer secret field, it depends on which one field is the last one been changed. So we should not add other info to this one. And if user set peer user, the peer secret field will be marked as required.

Comment (a-stjohn) [reply src/app/helptext/sharing/iscsi/iscsi.ts]: ahh. Sorry. I misunderstood what was happening in the UI. In that case, it looks good to me!

Comment (lola27) [reply src/app/helptext/sharing/iscsi/iscsi.ts]: thanks @a-stjohn ! The name is a little bit confused, so I pushed a commit to changed it.

**Pull request merged**